samples/snippets/src/main/java/search/SearchSimpleQuery.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2021 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * [START retail_search_for_products_with_query_parameter]
+ * Call Retail API to search for a products in a catalog using only search query.
+ */
+
+package search;
+
+import com.google.cloud.retail.v2.SearchRequest;
+import com.google.cloud.retail.v2.SearchResponse;
+import com.google.cloud.retail.v2.SearchServiceClient;
+import com.google.cloud.retail.v2.SearchServiceSettings;
+import java.io.IOException;
+import java.util.UUID;
+
+public class SearchSimpleQuery {
```

Comment:
Please make sure these samples follow the [sample style guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)

Reply:
checked